README.md

```diff
@@ -558,6 +566,17 @@ Pebble.addEventListener('webviewclosed', function(e) {
 
 ### `Clay([Array] config, [function] customFn, [object] options)`
 
+#### Properties
+
+| Property | Type | Description |
+|----------|------|-------------|
+| `.config` | Array | Reference to the config passed to the constructor and used for generating the page. **WARNING** this is a direct reference, not a copy of the config so any modification you make to it, will be reflected on the original as well |
```

Comment:
Does a change to the original mean JS code can change the page actually being shown while it's open? Or does it mean it will overwrite the bundled app resource (which I thought were read-only)?